OneSignalSDK/unittest/src/test/java/com/test/onesignal/TestHelpers.java

```diff
@@ -104,8 +114,6 @@ static void beforeTestInitAndCleanup() throws Exception {
 
       ShadowDynamicTimer.resetStatics();
 
-      ShadowOSWebView.resetStatics();
```

Comment:
This removal doesn't look intentional. Can we put it back?

Reply:
It was a duplicate

Reply:
ah you are right